src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -28,7 +29,13 @@
 }
 
 .HeroRecommendation-info {
-  @include margin-start(24px);
+  display: flex;
+  flex-direction: column;
+  min-width: 0;
```

Comment:
I don't see this overriding anything. Is it still needed? When I remove it, the page still appears fixed when using the STR in the bug report.

Reply:
You're right, this doesn't seem to be needed. I guess it was an artifact of some experimenting I was doing.